mamba_package/src/package.cpp

```diff
@@ -35,6 +36,10 @@ set_package_command(CLI::App* com)
         "-c,--compression-level",
         compression_level,
         "Compression level from 0-9 (tar.bz2, default is 9), and 1-22 (conda, default is 15)");
+    compress_subcom->add_option(
+        "--compression-threads",
+        compression_threads,
+        "Compression threads (only relevant for .conda packages, default is 1)");
```

Comment:
I don't know if zstd does "one thread if 1, a single thread but doesn't use threads at all if 0" or -1 to automatically choose threading based on processors